teuthology/task/clock.py

```diff
@@ -8,6 +8,9 @@
 
 log = logging.getLogger(__name__)
 
+def filter_cluster(cluster):
```

Comment:
usage of this function would be more obvious in context if it referred to 'container' somehow

Reply:
True - or, actually, I don't even know why I thought it would be important to abstract this into a separate function.

Reply:
maybe you thought about making it take arguments to specify how to filter.  but it's probably overkill

Reply:
Looks like I abstracted it because the same logic is used a few times throughout the task. Added a docstring.

Reply:
I still think there's value in the name expressing more of its purpose so that the call site is more self-documenting, but it's a pretty small nit.

Reply:
You think something like `filter_cluster_no_containers` would be better?

Reply:
yeah, or filter_out_containers(), or remove_containers() (the cluster is the arg, so)

Reply:
Done!